serverless-sample/app/aws/Makefile

```diff
@@ -0,0 +1,6 @@
+
+build:	
+	CGO_ENABLED=0 GOOS=linux go build -v -installsuffix nocgo -o .builds/binary-aws main.go
+upload:	
+	docker build -t cloudrun-server-aws:latest .
+	docker tag cloudrun-server-aws:latest <ecr-address>/runner-demo:latest
```

Comment:
These steps seem inconsistent with the GCP version - I think they should be the same
Also, I didn't think that "docker tag" actually pushes the image. Don't you also need a docker push command here?

See my Makefile comments on the gcp version.

Reply:
fixed